### PR TITLE
adds debitCardAllowed field to CreateCardManagementPageInitializeRequest

### PR DESCRIPTION
**Pull request by mustafacantekir**
adds debitCardAllowed field to CreateCardManagementPageInitializeRequest

### CODE REVIEW

File: src/main/java/com/iyzipay/request/CreateCardManagementPageInitializeRequest.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
     private Boolean addNewCardEnabled;
     private Boolean validateNewCard;
+    private Boolean debitCardAllowed;
```
Comment (AlicanAkkus) [inline src/main/java/com/iyzipay/request/CreateCardManagementPageInitializeRequest.java]: this field can lead to a fault in hash computation. should be added to toString() method.

Comment (mustafacantekir) [reply src/main/java/com/iyzipay/request/CreateCardManagementPageInitializeRequest.java]: done

File: src/main/java/com/iyzipay/request/CreateCardManagementPageInitializeRequest.java [hunk 2]
```diff
@@ -67,6 +76,7 @@ public String toString() {
                 .appendSuper(super.toString())
                 .append("addNewCardEnabled", addNewCardEnabled)
                 .append("validateNewCard", validateNewCard)
+                .append("debitCardAllowed", debitCardAllowed)
```
Comment (AlicanAkkus) [inline src/main/java/com/iyzipay/request/CreateCardManagementPageInitializeRequest.java]: this field should be added to the end of append.

Comment (mustafacantekir) [reply src/main/java/com/iyzipay/request/CreateCardManagementPageInitializeRequest.java]: done

**Pull request merged**